accepted/0015-virtual-camera-support.md

```diff
@@ -0,0 +1,42 @@
+- Start Date: 2020-03-25
+- RFC PR: #15
+- Related Github Issue: https://github.com/obsproject/obs-studio/issues/25685
+
+# Summary
+
+Add the ability to render output to a virtual camera device on Linux, Mac, and Windows.
+
+# Motivation
+
+OBS is a powerful set of tools to manipulate live video streams that natively supports output to popular streaming services as well as rendering to a local video file. There are a huge number of people who engage in 1:1 or small-scale streaming using video conferencing software like Zoom or Google Hangouts. Many of these people have similar needs to those of traditional streamers, but for one reason or another cannot switch the video conferencing software they use (social graph, corporate policy). One way to deliver OBS output to these applications is to provide a virtual camera in OBS.
+
+Similar functionality: 
+* OBS can be extended by the [OBS-VirtualCam plugin](https://obsproject.com/forum/resources/obs-virtualcam.539/). Currently, only Windows is supported.
```

Comment:
Note: a relatively similar plugin is also available for Linux, called [obs-v4l2sink](https://github.com/CatxFish/obs-v4l2sink), made by the same developer. Note, however, it requires another component, like v4l2loopback, to actually output to another application to the system. No such plugin is currently available on macOS.